mautic/mautic
# New option to reset fields with campaign actions: Update contact and Update contact's primary company

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | 
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | https://github.com/mautic/mautic/issues/6553
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:

With Update contact/Update contact's primary company campaign action cannot  reset contact fields values at the moment. This update bring new option: **Update fields with empty value**. Selected fields will be update with empty values

![image](https://user-images.githubusercontent.com/462477/45560987-e42a5080-b846-11e8-9bca-2340efe67680.png)


#### Steps to test this PR:
1. Create contact with firstname and with primary company with website.
2.  Create campaign with 
- Update contact action with  **Update fields with empty value** for firstname
- Update contact's primary company action with  **Update fields with empty value** for website
3. Run campaign  and see If firstname for contact and website for company was reset (set to null)
